pkgs/tools/misc/pastel-rs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  name = "pastel-rs";
+  pname = "pastel";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sharkdp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0f54p3pzfp7xrwlqn61l7j41vmgcfph3bhq2khxh5apfwwdx9nng";
+  };
+
+  cargoSha256 = "05yvlm7z3zfn8qd8nb9zpch9xsfzidrpyrgg2vij3h3q095mdm66";
+
+  meta = with stdenv.lib; {
+    description = "A rust command-line tool to generate, analyze, convert and manipulate colors";
+    homepage = "https://github.com/sharkdp/pastel";
+    license = with licenses; [ mit apache ];
+    maintainers = [ maintainers.mtrsk ];
```

Comment:
Thanks for making yourself a maintainer! This currently isn't building though because mtrsk is currently not a maintainer on the master branch. Could you add yourself to `maintainers/maintainer-list.nix`?

Reply:
Can the addition to `maintainer-list.nix` be in a separate commit which happens before the current one and titled `maintainers: add wtrsk`

Reply:
@mtrsk Could you rewrite the git history so your pull request has only two commits? That way, there is only one "init" commit and all your commits will safely build